### ISSUE TITLE
Bug: Add missing language attribute to the application

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#3847]
The `<html>` tag requires a [language attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang). As the application is currently only in English, hardcode `lang="en"` to the HTML tag.

The file and line to fix is https://github.com/open-sauced/app/blob/beta/pages/_document.tsx#L5

```diff
import { Html, Head, Main, NextScript } from "next/document";

export default function Document() {
  return (
-    <Html style={{ "--top-nav-height": "3rem" }}>
+    <Html lang="en" style={{ "--top-nav-height": "3rem" }}> 
      <Head />
      <body className="selection:bg-light-orange-8 selection:text-light-slate-12">
        <Main />
        <NextScript />
      </body>
    </Html>
  );
}

```

**Comment by AlexVCS** [comment]
.take
